### PR TITLE
Factor out class BlocksPanel

### PR DESCRIPTION
**Pull request by samreid**
Factor out class BlocksPanel for https://github.com/phetsims/density-buoyancy-common/issues/128

### CODE REVIEW

**Review by AgustinVallejo — CHANGES_REQUESTED** [review]
Really nice changes! ✨Can you add a line of comment saying hi?

**Review by zepumph — APPROVED** [review]
I love that there are no comments saying "hi". Thanks for doing that.

File: js/buoyancy/view/BuoyancyIntroScreenView.ts [hunk 1]
```diff
@@ -182,7 +166,7 @@ export default class BuoyancyIntroScreenView extends DensityBuoyancyScreenView<B
 
       cuboidPDOMLayer,
 
-      blocksRadioButtonGroup,
+      blockSetPanel,
```
Comment (AgustinVallejo) [inline js/buoyancy/view/BuoyancyIntroScreenView.ts:169]: Why PDOM order what's this? omg

File: js/buoyancy-basics/view/BuoyancyBasicsCompareScreenView.ts
Comment (AgustinVallejo) [inline js/buoyancy-basics/view/BuoyancyBasicsCompareScreenView.ts:1]: What a nice file!

File: js/density/view/DensityCompareScreenView.ts [hunk 2]
```diff
@@ -7,22 +7,19 @@
  */
 
 import Vector3 from '../../../../dot/js/Vector3.js';
-import { AlignBox, Node, Text, VBox } from '../../../../scenery/js/imports.js';
-import Panel, { PanelOptions } from '../../../../sun/js/Panel.js';
-import VerticalAquaRadioButtonGroup from '../../../../sun/js/VerticalAquaRadioButtonGroup.js';
+import { AlignBox, Node } from '../../../../scenery/js/imports.js';
```
Comment (AgustinVallejo) [inline js/density/view/DensityCompareScreenView.ts:10]: Non default imports are nice

**Review by AgustinVallejo — COMMENTED** [review]
Done with the second review

**Review by AgustinVallejo — CHANGES_REQUESTED** [review]
I insist on the comment saying "hi" !!!!!!!!!!

**Review by samreid — COMMENTED** [review]
OK I addressed the requests

**Review by AgustinVallejo — APPROVED** [review]
I yield

**Pull request merged**